handlers/base.go

```diff
@@ -0,0 +1,18 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+func RespondWithJSON(w http.ResponseWriter, code int, payload interface{}) {
+	response, _ := json.Marshal(payload)
+
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(code)
+	w.Write(response)
+}
+
+func RespondWithError(w http.ResponseWriter, code int, message string) {
```

Comment:
is this needed?

Reply:
IMO, this is a must for any API app. We should standardize the output in order to get things easier to API consumers. If we respond errors in multiple formats, UI can get crazy.
If we are not adding in this PR, I most probably add it in the next one. For example, in the following line https://github.com/swift-sunshine/swscore/blob/master/handlers/root.go#L18 which instead of responding with exception, we want to channel it into a standardized json.
Same in line L#23, L#29.